src/rail/estimation/algos/deepdisc.py

```diff
@@ -168,6 +167,13 @@ class DeepDiscInformer(CatInformer):
         output_name=Param(str, "deepdisc_informer", required=False, msg="What to call the generated output."),
         chunk_size=Param(int, 100, required=False, msg="Chunk size used within detectron2 code."),
         training_percent=Param(float, 0.8, required=False, msg="The fraction of input data used to split into training/evaluation sets"),
+        num_camera_filters=Param(int, 6, required=False, msg="The number of camera filters for the dataset used (LSST has 6)."),
+        epochs_per_print=Param(int, 5, required=False, msg="How often to print in-progress output."),
+        head_iters=Param(int, 0, required=False, msg="How many iterations in a head run."),
```

Comment:
My grasp of heads and backbones is not quite strong enough to write a meaningful description here - I'm all ears for something better for this

Reply:
head_iters: used when training with a frozen backbone (then, we go and unfreeze the backbone, and run everything)